grasp_service:
  ros__parameters:
    joints_names:
      - joint1
      - joint2

    positions_to_target_list:
      - FirstPostion

    positions_to_target:
      FirstPostion: [0.0, 0.5]

    times_for_positions_to_target:
      FirstPostion: 5

    positions_to_home_list:
      - FirstPostion

    positions_to_home:
      FirstPostion: [0.0, -0.3]

    times_for_positions_to_home:
      FirstPostion: 10

    time_to_wait_on_target: 10